docs/service_controller.md

```diff
@@ -3,33 +3,33 @@
+| service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy            | -                                   | ELBSecurityPolicy-2016-08 | ELB,NLB | Specifies SSL negotiation settings for the HTTPS/SSL listeners of your load balancer. Defaults to the default ELB policy.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| service.beta.kubernetes.io/aws-load-balancer-ssl-ports                         | Comma-separated list                | *                         | ELB,NLB | Specifies a comma-separated list of ports that will use SSL/HTTPS listeners. Defaults to all.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| service.beta.kubernetes.io/aws-load-balancer-type                              | [nlb]                               | -                         | ELB,NLB | Indicates the type of Load Balancer. The only valid value is `nlb`, this means that leaving this field blank or omitting the annotation is equivalent to selecting ELB. When selecting `nlb`, the backend protocol is automatically derived from the protocol defined in the Kubernetes Service, as long as that protocol is supported by NLB (supported protocols include TCP, the default for a Kubernetes Service, and UDP). If the protocol is TCP and the `aws-load-balancer-ssl-cert` annotation is also present, the NLB protocol will be set to TLS                                                                         |
```

Comment:
nit
```suggestion
| service.beta.kubernetes.io/aws-load-balancer-type                              | [nlb]                               | -                         | ELB,NLB | Indicates the type of Load Balancer. The only valid value is `nlb`, this means that leaving this field blank or omitting the annotation is equivalent to selecting ELB. When selecting `nlb`, the backend protocol is automatically derived from the protocol defined in the Kubernetes Service, as long as that protocol is supported by NLB (supported protocols include TCP, the default for a Kubernetes Service, and UDP). If the protocol is TCP and the `aws-load-balancer-ssl-cert` annotation is also present, the NLB protocol will be set to TLS.                                                                        |
```

Reply:
looks like the period got dropped from the latest commit.